index.html

```diff
@@ -473,6 +473,38 @@ <h3>Importing retro toys</h3>
         import of pirated product.</p>
       <p class="shortUcContributor">Suggested by DHS SVIP</p>
     </section>
+
+    <section id="didcomm">
+      <h3>DIDComm</h3>
+      <p>
+        DIDComm is a low-level protocol for interoperable, authenticated, 
+        end to end encryption using decentralized identifiers. 
+        It is transport-agnostic, meaning it can run over HTTP(S), 
+        email, sockets, Bluetooth, AMQP, sneakernet, NFC, XMPP, Kafka, 
+        or any other mechanism that makes data flow. 
+        It is message-based and asynchronous. 
+        DIDComm can be used as a foundation for any high-level protocol 
+        exchange sensitive information. DIDComm is not API-centric, 
+        which means it doesn't need a different serviceEndpoint for each protocol.
+      </p>
+
+      <p>
+        Some examples of problems solved by DIDComm-based protocols 
```

Comment:
What is the goal for having these examples?  I'm trying to understand why these ones were selected, and particularly the last two.  Random or was there some goal in picking these ones vs. others?

Reply:
I picked them for variety. The obvious ones about issuing and proving are in the list; the rest are just examples of how to solve interesting problems.

Reply:
We also wanted some examples that implied or used the "service" data model in the did document... because we think its fundamental to communication.... in other words, didcomm is one of the strongest reasons to define services from an interop perspective.... this is relevant since there is discussion about their privacy implication / removing them.